src/main/java/io/cdap/plugin/dlp/Utils.java

```diff
@@ -241,14 +241,14 @@ public static Table getTableFromStructuredRecord(StructuredRecord record) throws
             break;
           case INT:
           case LONG:
-            valueBuilder.setIntegerValue((Long) fieldValue);
+            valueBuilder.setIntegerValue(Long.parseLong(fieldValue.toString()));
```

Comment:
this should just be `(long) fieldValue)`

Reply:
assuming fieldValue cannot be null here

Reply:
oh...I dont know why I though it needed the non-primitive type. Done